services/motion/builtin/utils.go

```diff
@@ -0,0 +1,532 @@
+package builtin
```

Comment:
can we call this `testutils` to connote that this file is only really used in testing?

Reply:
Yep, will rename